dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,33 +1,45 @@
 -# English only for now
 - if language == "en"
 
-  -# Never show the survey.
-  - if false
+  -# US only
+  - if request.location.try(:country_code) == 'US' || (!Rails.env.production? && request.location.try(:country_code) == 'RD')
 
     -# If signed in, and user hasn't already filled out this survey
-    - if current_user && !current_user.survey2015_value
+    - if current_user && true # !current_user.survey2016_value
```

Comment:
Don't we want to only be prompting them if they haven't already filled out the survey?